package.json

```diff
@@ -1,13 +1,15 @@
 {
-  "name": "JSExercises",
+  "name": "js-exercises",
   "version": "1.0.0",
   "description": "I use [VS Code](https://code.visualstudio.com/) to edit code, [Prettier](https://prettier.io/) for formatting",
-  "main": "index.js",
+  "main": "build/index.js",
```

Comment:
i don't think you have a index.js entry point, lets remove this kv pair from the file

Reply:
what's a kv pair

Reply:
key-value